backend/src/database/entities/base.entity.ts

```diff
@@ -0,0 +1,9 @@
+import { CreateDateColumn, UpdateDateColumn } from "typeorm";
+
+export class BaseEntity {
+  @CreateDateColumn({ name: 'created_at' })
```

Comment:
Should we include id column here as well?

Reply:
No, because I'm switching between uuid (personnel) and incremental id (other things)